### DISCUSSION TITLE
Oracle DB Plugin

### DISCUSSION THREAD

**Discussion by nazakahar** [dbgate/dbgate#658] [Q&A]
Does anybody have any guide on how to get the setup to work especially in Docker to use oracle db plugin with dbgate? I tried but then I got an error as follows:

{"pid":1206,"caller":"dbconnProcess","name":"databaseConnection","err":{},"level":50,"msg":"Error in DB connection","time":1699493484032}

after it ran the following sql statement:
SELECT version as \"version\" FROM v$instance","time":1699493484034}

It would be useful if dbgate has something in the log that says connection is succesful so that we are aware where to troubleshoot.

**Comment by LawrenceJiang** [comment]
You may install the new version:5.30.
